app/helpers/home_page_helper.rb

```diff
@@ -0,0 +1,2 @@
+module HomePageHelper
```

Comment:
Is this file used at all?

Reply:
I think i will actually use it today :))